hles/extraction/src/main/scala/org/broadinstitute/monster/dap/EnvironmentExtractionPipeline.scala

```diff
@@ -22,6 +22,7 @@ object EnvironmentExtractionPipeline extends ScioApp[Args] {
   )
 
   val subdir = "environment"
+
```

Comment:
getting a bit worried about the churn on this line - is this just because we haven't merged any of the multiple PRs that `scalafmt` made this change in, or will we continue to see diffs on this line in future PRs?

Reply:
Good eye, I think it's because we have yet to merge any of the multiple PRs.